.github/pull_request_template.md

```diff
@@ -1,68 +1,33 @@
-<!--
+## 🎫 Ticket
 
---- PR Hygiene Checklist ---
+https://jira.cms.gov/browse/BCDA-xxx
 
-1. Make sure your branch is named with this format: `user-initials/description-ABC-123`. For example, `jj/add-awesomeness-bcda-99999`
-2. Update the PR title: `bcda-99999 Feature: Add Awesomeness`
-3. Edit the text below - do not leave placeholders in the text.
-4. Add any other details that will be helpful for the reviewers: details description, screenshots, etc
-5. Request a review from someone/multiple someones
--->
+## 🛠 Changes
 
-<!-- Replace xxx with the JIRA ticket number: -->
+(What was added, updated, or removed in this PR.)
 
-### Fixes [BCDA-xxx](https://jira.cms.gov/browse/BCDA-xxx)
+## ℹ️ Context for reviewers
 
-<!-- Describe the problem being solved here: -->
+(Background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers.)
```

Comment:
I'm not really sure if we track this kind of thing, but would it be useful to anyone on our team if we included PR "type" anywhere in the PR context? Ex: bug, feature, hotfix, breaking change, etc. 

If that would be something better suited elsewhere like the branch naming conventions, that could be an option too, if this is something that we'd like to consider! Ex. feature/BCDA-1234, bugfix/BCDA-1000

Reply:
yeah that's a good question! we could maybe use github labels as a starting point? I know some teams also like to indicate them in the commit messages but that's a larger change that I'd like to consider more closely since it's harder to enforce (can't change the commit message after merging!)

Reply:
that sounds good to me! If it's something that would be useful, we can experiment with the labels.